src/iarc7_motion/iarc_tasks/track_roomba_task.py

```diff
@@ -222,7 +224,7 @@ def _check_max_roomba_range(self):
         _distance_to_roomba = math.sqrt(self._roomba_point.point.x**2 + 
                             self._roomba_point.point.y**2)
         
-        return (_distance_to_roomba <= self._MAX_START_TASK_DIST + self._overshoot)
+        return (_distance_to_roomba <= self._MAX_START_TASK_DIST + ((self._x_overshoot)**2 + (self._y_overshoot)**2)**(0.5))
```

Comment:
I think this is over 80 characters

Reply:
Also I don't think this is the condition we want here.  Most likely this should stay like it was before, as a max distance from the center of the roomba regardless of the specified offset.  If this is intended to be a cap on the initial distance from the target point, it should be a max distance to the center of the roomba plus the offset, which is a little different from what it's doing now